tests/unit/info.tcl

```diff
@@ -384,6 +384,14 @@ start_server {tags {"info" "external:skip" "debug_defrag:skip"}} {
         }
 
         test {stats: client input and output buffer limit disconnections} {
+            # Disable copy avoidance
+            set min_threads [lindex [r config get min-io-threads-avoid-copy-reply] 1]
+            set min_size [lindex [r config get min-string-size-avoid-copy-reply] 1]
+            set min_size_threaded [lindex [r config get min-string-size-avoid-copy-reply-threaded] 1]
+            r config set min-io-threads-avoid-copy-reply 0
+            r config set min-string-size-avoid-copy-reply 0
+            r config set min-string-size-avoid-copy-reply-threaded 0
+
```

Comment:
Same question, why are we disabling copy avoidance here?

Reply:
This test relies on `closeClientOnOutputBufferLimitReached`, which doesn't trigger disconnections because `c->reply_bytes` is `0`.

Reply:
Let's document that I suppose. 

> offloaded data is not counted for cob usage

Do we ever kill the clients for CoB usage if it's just a bunch of offloaded replies?